example/app/source/Application.h

```diff
@@ -18,7 +18,7 @@ class Application final : public juce::JUCEApplication
         return "0.0.0";
     }
 
-    void initialise (const juce::String &) override
+    void initialise ([[maybe_unused]] const juce::String & commandLineArguments) override
```

Comment:
why this isn't `may_be_unused` or `permit_unused` is beyond me

Reply:
The [use of underscores](https://en.cppreference.com/w/cpp/language/attributes) is fairly inconsistent too (`[[nodiscard]]` and `[[maybe_unused]]`) 🤷🏼‍♂️